snap/snapcraft.yaml

```diff
@@ -0,0 +1,44 @@
+name: smartctl-exporter
+base: core24
+adopt-info: smartctl_exporter
+summary: Export smartctl statistics to Prometheus
+license: Apache-2.0
+contact: solutions-engineering@lists.canonical.com
+issues: https://github.com/canonical/smartctl-exporter-snap/issues
+source-code: https://github.com/canonical/smartctl-exporter-snap
+description: |
+  This snap includes smartctl_exporter to provide smartctl statistics to Prometheus.
+grade: stable
+confinement: strict
+platforms:
+  amd64:
+  arm64:
+
+apps:
+  smartctl_exporter:
```

Comment:
@chanchiwai-ray looks like we also need to update this as well, and probably the `parts.smartctl_exporter` name too.